site2/docs/client-libraries-java.md

```diff
@@ -208,6 +208,7 @@ Name| Type |  <div style="width:300px">Description</div>|  Default
 `batchingMaxMessages` |int|The maximum number of messages permitted in a batch.|1000
 `batchingEnabled`| boolean|Enable batching of messages. |true
 `compressionType`|CompressionType|Message data compression type used by a producer. <br />Available options:<li>[`LZ4`](https://github.com/lz4/lz4)</li><li>[`ZLIB`](https://zlib.net/)<br /><li>[`ZSTD`](https://facebook.github.io/zstd/)</li><li>[`SNAPPY`](https://google.github.io/snappy/)</li>| No compression
+`initialSubscriptionName`|string|Use this config to automatically create an initial subscription when creating the topic. If this field is not set, the initial subscription will not be created.|null
```

Comment:
```suggestion
`initialSubscriptionName`|string|Use this configuration to automatically create an initial subscription when creating a topic. If this field is not set, the initial subscription is not created.|null
```

Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true. Details: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp

Reply:
Thanks for your suggestions.